contrib/debian/control

```diff
@@ -189,7 +189,7 @@ Depends: dkms (>> 2.1.1.2-5),
          file,
          libc6-dev | libc-dev,
          lsb-release,
-         python3-distutils | libpython3-stdlib (<< 3.6.4),
+         python3 (>> 3.12) | python3-distutils | libpython3-stdlib (<< 3.6.4),
```

Comment:
From Python documentation, `sysconfig` module is added in stdlib from version 3.2. Maybe `>> 3.2` is enough so that it no longer pulls in `distutils` when `sysconfig` is present?

Reply:
That's a good thought, but I'm not really set up to test it currently so let's leave that for another PR.